test/integration/simple-deployment-evaluation/00-assert.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test
+  name: test
+status:
+  readyReplicas: 1
```

Comment:
Given the comment on the install.yaml, I suggest testing for >1 replicas here to verify the HPA kicked in

Reply:
that should be covered in a separate test